tests/test_core.py

```diff
@@ -1475,70 +1475,10 @@ def test_bigarray(self):
   def test_mod_globalstruct(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_mod_globalstruct')
 
-  @no_wasm_backend()
+  @no_wasm_backend('specific alignment semantics may be asmjs-specific?')
```

Comment:
Hmm, this test doesn't use long doubles which are the only point I can think of where struct alignment can differ? So either there is another struct alignment factor, or something else is going wrong here.

Note that one thing this tests is that `generations` initializer, which is self-referential. That's my bet here...